src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -27,6 +27,12 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; 'handler' functions for HTTP endpoints
 
+(defn handle-server-error
```

Comment:
This looks similar in some ways to https://github.com/puppetlabs/puppet-server/blob/0f8cd3602517477a02140bfd961de6a785800a04/src/clj/puppetlabs/puppetserver/jruby_request.clj#L9-L13, which throws a slingshot error that triggers a 400 response. I wonder if we should be consolidating these in some way.